server/src/main/java/org/apache/iotdb/db/mpp/buffer/DataBlockService.java

```diff
@@ -38,8 +50,25 @@ public ThriftService getImplementation() {
   @Override
   public void initTProcessor()
       throws ClassNotFoundException, IllegalAccessException, InstantiationException {
-    impl = new DataBlockServiceImpl();
-    processor = new Processor<>(impl);
+    executorService =
+        IoTDBThreadPoolFactory.newCachedThreadPoolWithDaemon("data-block-manager-task-executors");
```

Comment:
`newCachedThreadPoolWithDaemon`'s `corePoolSize` is `0` and `maximumPoolSize` is `Integer.MAX_VALUE`. It's better to use FixedCountThreadPool or set the `corePoolSize` and `maximumPoolSize` in `CachedThreadPool` to a reasonable value.

Reply:
Agree. But the `reasonable` value needs some further testing. What about setting the `corePoolSize` to 0 and `maximumPoolSize` to 10 for now?

Reply:
you can set both `corePoolSize` and `maximumPoolSize` to 4 and to make it configurable, you can define it in the `iotdb-engine.properties`.

Reply:
Fixed.